packages/renderer/src/lib/kube/details/ObjectMeta.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+import type { V1ObjectMeta } from '@kubernetes/client-node';
+import { rowStyle, titleStyle, internalKubernetesKeys } from './utils';
+import { faChevronDown, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+
+export let artifact: V1ObjectMeta | undefined;
+
+if (artifact?.creationTimestamp) {
+  artifact.creationTimestamp = new Date(artifact.creationTimestamp);
+}
+
+let internalLabelsDropdownOpen: boolean = false;
+let internalAnnotationsDropdownOpen: boolean = false;
+
+// Sort labels out to internal and external
+let labels: [string, string][] = [];
+let internalLabels: [string, string][] = [];
+if (artifact?.labels) {
+  Object.entries(artifact.labels).forEach(([key, value]) => {
+    if (internalKubernetesKeys.includes(key)) {
+      internalLabels.push([key, value]);
+    } else {
+      labels.push([key, value]);
+    }
+  });
+}
+
+// Sort annotationsout to internal and external
+let annotations: [string, string][] = [];
+let internalAnnotations: [string, string][] = [];
+if (artifact?.annotations) {
+  Object.entries(artifact.annotations).forEach(([key, value]) => {
+    if (internalKubernetesKeys.includes(key)) {
+      internalAnnotations.push([key, value]);
+    } else {
+      annotations.push([key, value]);
+    }
+  });
+}
```

Comment:
it feels weird that it's not inside onMount method ?

Reply:
So because it is simple sorting, I did not put it in the onMount method since I get this error:

```
if_block0.p is not a function
```

When svelte tries to check: `  {#if annotations.length > 0}`

It seems to stem from this issue: https://github.com/sveltejs/svelte/issues/5789